triple-lift.js

```diff
@@ -231,60 +250,81 @@ window.headertag.partnerScopes.push(function() {
 
         //? }
 
-        var yourBidder = new Partner(config);
-        window.headertag[PARTNER_ID] = {};
-        window.headertag[PARTNER_ID].callback = yourBidder.responseCallback;
-        window.headertag[PARTNER_ID].render = yourBidder.renderAd;
+        var tplBidder = new Partner(config);
+
+        try {
+            window.headertag[PARTNER_ID] = {};
+            window.headertag[PARTNER_ID].callback = tplBidder.responseCallback;
+            window.headertag[PARTNER_ID].render = tplBidder.renderAd;
```

Comment:
One more update here.

Our new render function format is:
`window.headertag.TripleLiftHtb.render` so can you please remove this line and replace it with the following:
```
        window.TripleLiftHtb = {};
        window.TripleLiftHtb.render = tplBidder.renderAd;
```